src/pykeen/models/base.py

```diff
@@ -69,7 +69,6 @@ def __init__(
         predict_with_sigmoid: bool = False,
         preferred_device: DeviceHint = None,
         random_seed: Optional[int] = None,
-        regularizer: Optional[Regularizer] = None,
```

Comment:
This wasn't actually relevant to the PR, but I noticed it was unused and shouldn't be there